src/Apps/Order/Components/__tests__/PartnerOfferTimerItem.jest.tsx

```diff
@@ -1,87 +1,56 @@
-import {
-  RelayEnvironmentProvider,
-  graphql,
-  useLazyLoadQuery,
-} from "react-relay"
-import { MockPayloadGenerator, createMockEnvironment } from "relay-test-utils"
-
-import { PartnerOfferTimerItemTestQuery } from "__generated__/PartnerOfferTimerItemTestQuery.graphql"
+import { graphql } from "react-relay"
 import { PartnerOfferTimerItem } from "Apps/Order/Components/PartnerOfferTimerItem"
-import { act, render, screen } from "@testing-library/react"
-import { Suspense } from "react"
-import { flushPromiseQueue } from "DevTools/flushPromiseQueue"
+import { screen } from "@testing-library/react"
 import { DateTime } from "luxon"
+import { setupTestWrapperTL } from "DevTools/setupTestWrapperTL"
```

Comment:
@narikazu - just a heads up that we have formal relay testing tools, and we never have to manually wire up data deps in our tests